CHANGES_NEXT_RELEASE

```diff
@@ -11,3 +11,4 @@
 - [FEATURE] Enable per-user or default Hive database creation in OrionHDFSSink(#516)
 - [HARDENING] Document the service_as_namespace OrionHDFSSink parameter (#692)
 - [HARDENING] Add a piece of documentation about multitenancy (#607)
+- [HARDENING] Add a piece of documentation about dependencies (#635)
```

Comment:
I would say "Add a piece of documentation about required dependencies to be manually installed when building and installing without dependencies"